smslibrary/src/main/java/com/eis/smslibrary/exceptions/InvalidTelephoneNumberException.java

```diff
@@ -1,35 +1,19 @@
 package com.eis.smslibrary.exceptions;
 
-
-import com.eis.smslibrary.SMSPeer;
-
 /**
- * Exception usually thrown if a Peer has been instantiated without having the telephone number checked for validity
+ * Exception usually thrown if a Peer has been instantiated without having the telephone number checked for validity.
```

Comment:
That point tho, lol

Reply:
Yes I'm a bit strict on these things. If a method gets my name it has to be homogenized with some rules.
It would be cool to have a styiling guidelines paper.